content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+
+```rust
+commands
+    .spawn_bundle(SceneBundle {
+        scene: asset_server.load("player.gltf#Scene0"),
+        transform: Transform::from_xyz(10.0, 20.0, 30.0),
+        ..default()
+    })
+    .insert(Player::default());
+```
+
+[`SceneBundle`]: https://docs.rs/bevy/0.8.0/bevy/scene/struct.SceneBundle.html
+
+## Parallel Frustum Culling
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+Bevy uses ["frustum culling"](/news/bevy-0-6/#visibility-and-frustum-culling) to skip drawing entities that are outside of the camera's view. In **Bevy 0.8**, frustum culling is now done in parallel. When culling thousands of entities, this yields significant performance improvements:
+
+### frustum culling system time vs number of entities culled (lower is better) 
+
+![parallel frustum culling](parallel_frustum_culling.png)
+
+Note that "parallel b" stand for "parallel batch size" (number of entities in each batch). We selected 1024 as our batch size in **Bevy 0.8**, as that performs better.
+
+## Automatic Mesh Tangent Generation
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump), @DJMcNab</div>
+
+Vertex tangents are used in tandem with normal maps to give meshes more detailed normals when rendering them. Some imported meshes have normal maps, but don't have vertex tangents calculated. Bevy can now automatically generate vertex tangents for [`Meshes`][`Mesh`] that are missing them using the defacto industry-standard MikkTSpace library / algorithm (Godot, Unity, Unreal, and Blender all use this).
+
+We have started maintaining [our own fork](https://github.com/bevyengine/bevy/tree/v0.8.0/crates/bevy_mikktspace) of the [gltf-rs/mikktspace crate](https://github.com/gltf-rs/mikktspace) so we can:
+1. Update dependencies at the speed required for Bevy
+2. [Start reining in the unsafe code](https://github.com/bevyengine/bevy/pull/4932) (it currently uses unsafe rust code auto-generated from the original mikktspace.h written in C).
+
+## Default to Linear Texture Filtering
+
+<div class="release-feature-authors">authors: @aevyrie, @cart</div>
+
+Images in Bevy now use linear texture filtering by default, which is more in line with the rest of the gamedev ecosystem (Unity and Godot both default to filtered textures). 
+
+This means that textures that require unfiltered pixels (such as "pixel art" sprites) must override this default, either per-image:
+
+```rust
+image.sampler_descriptor = ImageSampler::nearest();
+```
+
+or globally using the new [`ImageSettings`] resource:
+
+```rust
+app.insert_resource(ImageSettings::default_nearest())
+```
+
+With that, we get crisp pixel art:
+
+![sprite](sprite.png)
+
+[`ImageSettings`]: https://docs.rs/bevy/0.8.0/bevy/render/texture/struct.ImageSettings.html
+
+## New GlobalTransform Matrix Representation
```

Comment:
We should calling out non-Euclidean geometries as a fun "flavor" line at the end of this section; I know the handful of folks looking for this are going to be pumped.

Reply:
I think non-Euclidean geometry requires access to the full 4x4 matrix (`Mat4`), which could be a fairly easy change to make, but I think someone who has a use should push for that if they want it.

Reply:
Hmm yeah unless we've proven this is possible, I'm not comfortable calling this out.

Reply:
Ah yep. I'm on board.